### PR TITLE
Master

### PR DESCRIPTION
**Pull request by amyxie361**
Merge development changes (add dpr retriever and reader, change argument settings) with master (training)

### CODE REVIEW

File: bertserini/experiments/eval/evaluate_v1_cmrc.py [hunk 1]
```diff
@@ -17,6 +17,9 @@
 #from utils import init_logger
 #logger = init_logger("evaluation")
 
+#install punkt corpus
+nltk.download('punkt')
```
Comment (amyxie361) [inline bertserini/experiments/eval/evaluate_v1_cmrc.py:21]: should not be added to the code

File: bertserini/experiments/evaluate.py [hunk 2]
```diff
@@ -55,16 +55,17 @@ def get_score_with_results(eval_data, predictions, mu, dataset):
     return eval_result, answers
 
 
-def get_best_mu_with_scores(eval_data, predictions, mu_range, dataset, output_path):
+def get_best_mu_with_scores(eval_data, predictions, mu_range, dataset, output_path, standard="f1"):
```
Comment (amyxie361) [inline bertserini/experiments/evaluate.py:58]: standard => metric

File: bertserini/reader/bert_reader.py [hunk 3]
```diff
@@ -98,14 +96,19 @@ def predict(self, question: Question, contexts: List[Context]) -> List[Answer]:
...
                 unique_id = int(eval_feature.unique_id)
 
-                output = [to_list(output[i]) for output in outputs]
-                
-                start_logits, end_logits = output
-                result = SquadResult(unique_id, start_logits, end_logits)
+                output = [outputs[oname][i] for oname in outputs]
+                start_logits = outputs.start_logits[i]
+                end_logits = outputs.end_logits[i]
+                try:
```
Comment (amyxie361) [inline bertserini/reader/bert_reader.py:102]: if type(start_logits) == Tensor: ...
